test/LibraryManager.Test/Configuration/SettingsTest.cs

```diff
@@ -143,6 +143,23 @@ public void TryGetValue_ValueExistsInDifferentCase_ReturnFalse()
             Assert.IsFalse(success);
         }
 
+        [TestMethod]
+        public void TryGetValue_IfSettingPresentAmongEnvironmentVariables_UseEnvironmentValue()
+        {
+            string fileText = @"{ ""config"": { ""testKey"": ""fileValue"" } }";
+            File.WriteAllText(TestFilePath, fileText);
+
+            Environment.SetEnvironmentVariable("testKey", "envValue");
```

Comment:
No concern about using a real environment variable for proxy in the tests versus some type of fake?

Reply:
Mostly a matter of hope ;)  And it's not limited to this new test, if the environment variable exists, it will impact all of the other tests in this class.

I thought about abstracting something like `IEnvironment`, but that complicated how the Settings get constructed... and we already have concrete dependencies on the file system in these tests, and making it pure quickly turned into a much larger change than I wanted to make now.